db/migrate/20190912121620_create_conversations.rb

```diff
@@ -0,0 +1,12 @@
+class CreateConversations < ActiveRecord::Migration[5.2]
+  def change
+    create_table(:conversations, &:timestamps)
+
+    create_table :conversation_users, bulk: true, id: false do |t|
```

Comment:
id: false ?

Reply:
I know 😢 I messed it up, I thought I wouldn't need that ID, since I could manage with (user_id, conversation_id). BUT NO
I arrange this here #17 , even ask you for forgiveness 🌝